frontend/src/container/MetricsApplication/Tabs/util.ts

```diff
@@ -48,20 +51,22 @@ export function onViewTracePopupClick({
 		const urlParams = new URLSearchParams(window.location.search);
 		urlParams.set(QueryParams.startTime, currentTime.toString());
 		urlParams.set(QueryParams.endTime, tPlusOne.toString());
+		urlParams.delete('relativeTime');
 		const avialableParams = routeConfig[ROUTES.TRACE];
 		const queryString = getQueryString(avialableParams, urlParams);
 
 		const JSONCompositeQuery = encodeURIComponent(
 			JSON.stringify(apmToTraceQuery),
 		);
 
-		const newTraceExplorerPath = `${
-			ROUTES.TRACES_EXPLORER
-		}?${urlParams.toString()}&selected={"serviceName":["${servicename}"]}&filterToFetchData=["duration","status","serviceName"]&spanAggregateCurrentPage=1&selectedTags=${selectedTraceTags}&${
+		const basePath = isViewLogsClicked
+			? ROUTES.LOGS_EXPLORER
+			: ROUTES.TRACES_EXPLORER;
+		const newPath = `${basePath}?${urlParams.toString()}&selected={"serviceName":["${servicename}"]}&filterToFetchData=["duration","status","serviceName"]&spanAggregateCurrentPage=1&selectedTags=${selectedTraceTags}&${
 			QueryParams.compositeQuery
 		}=${JSONCompositeQuery}&${queryString.join('&')}`;
 
-		history.push(newTraceExplorerPath);
+		history.push(newPath);
```

Comment:
can you make this reusable? since, IMO, we have written the same thing in https://github.com/SigNoz/signoz/blob/6685482ea687f9bd9c5691438e5b50ab09a05285/frontend/src/container/MetricsApplication/Tabs/Overview.tsx#L219-L227 as well?